topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,723 @@
+```
+
+> ### {% icon code-out %} Output
+> ```bash
+> <xarray.Dataset>
+> Dimensions:     ()
+>  Data variables:
+>     pm2p5_conc  float32 9.118
+>```
+{: .code-out}
+
+
+> ### {% icon question %} Maximum PM2.5 over Italy
+>
+> What is the maximum forecasted PM2.5 value over the Rome-Naples region?
+>
+> > ### {% icon solution %} Solution
+> >  We  select the same sub-area: 11. East to 15.0 East and 40. N to 43. N and compute the maximum with `max`. The maximum PM2.5 value is **59.13694382** μm.m-3 (that is rounded to **59.14**).
+> >
+> > > ### {% icon code-in %} Input: Python
+> > > ```python
+> > > dset.sel(latitude=slice(43., 40.), longitude=slice(11.,15.)).max()
+> > > ```
+> > {: .code-in}
+> > 
+> > > ### {% icon code-out %} Output
+> > > ```bash
+> > > xarray.Dataset
+> > > Dimensions:
+> > > Coordinates: (0)
+> > > Data variables:
+> > > pm2p5_conc
+> > > ()
+> > > float64
+> > > 59.14
+> > > array(59.13694382)
+> > > Attributes: (0)
+> > > ```
+> > {: .code-out}
+> {: .solution }
+{: .question }
+
+
+
+
+
+> ### {% icon question %} Find when the maximum PM2.5 is forecasted
+>
+> When is forecasted the maximum PM2.5 value?
+>
+> > ### {% icon solution %} Solution
+> > We will select a sub-area: 11. East to 15.0 East and 40. N to 43. N and average over the entire selected area and search where the maximum PM2.5 value of 59.13694382 μm.m-3 is found. The maximum PM2.5 value occurs on 2021-12-22 at 20:00 UTC.
+> >
+> > > ### {% icon code-in %} Input: Python
+> > > ```python
+> > > dset_tmean = dset.sel(latitude=slice(43., 40.), longitude=slice(11.,15.)).max(dim=('latitude', 'longitude'))
+> > > dset_tmean_max = dset_tmean.where(dset_tmean['pm2p5_conc'] == 59.13694382, drop=True)
+> > > print(dset_tmean_max)
+> > > ```
+> > {: .code-in}
+> > 
+> > > ### {% icon code-out %} Output
+> > > ```bash
+> > > <xarray.Dataset>
+> > > Dimensions:     (time: 1, level: 1)
+> > > Coordinates:
+> > >   * level       (level) float32 0.0
+> > >   * time        (time) timedelta64[ns] 20:00:00
+> > > Data variables:
+> > >     pm2p5_conc  (time, level) float32 59.14
+> > > ```
+> > {: .code-out}
+> {: .solution }
+{: .question }
+
+> ### {% icon comment %} Pixel size when averaging
+> We average over a relatively small so we do not make a weighted average. Use weighted averages when averaging over the entire globe or over a large area where the pixel sizes may vary (depending on the latitude).
+>
+{: .comment}
+
+
+## Resample
+- We often want to resample (on time dimension) our data:
+    - if your resampling frequency is lower than your original data, you would need to apply an operation on the data you group together, such as mean, min, max;
+    - if your resampling frequency is higher than your original data, you would need to indicate how to fill the gaps, for instance interpolate and indicate which interpolation method to apply or select nearest values, etc.
+
+### 1 day Resampling
```

Comment:
Perhaps it could be included inside the code block.

Reply:
you mean as a comment in the code?

Reply:
Yes, or perhaps fourth-level header (####)? Otherwise it is fine.

Reply:
I move each "comment" to the corresponding section (1 day reseampling and 30 mn resampling). I hope it makes it clearer.